TOKEN_GUIDE.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Build a Token Generator with Astro
+description: Learn how you can generate tokens using SSR frameworks like Astro.
+---
+
+I love building websites with Astro. It's my favorite web framework, and I don't want to use any other one. However, I must stress test it and ensure Astro is flexible enough to fully commit.
+
+Astro is created to be the perfect use case for a content-heavy website. The opposite of that is a backend, and in this guide, we will build a backend token generator to be used for Agora video calls.
+
+Spoilers. My love for Astro has grown deeper. 
+
+![Token Flow](/assets/token-flow.png)
+
+## Prerequisites
+1. NodeJS and Astro installed.
+2. A developer account with [Agora](https://console.agora.io/).
+
+## Project Setup
+To create a new Astro project, run `npm create astro@latest`. Follow all the default options during the setup process, except use an empty template. This way, our project will be free of distractions. Then, we must install the Agora Token package to generate our tokens. You can do that using `npm i agora-token`.
+
+You must also create a `.env` and add `APP_ID` and `APP_CERTIFICATE` values, which you can find within your Agora Console.
+
+## What is a Token?
+Tokens are a form of authentication that verifies whether the user can access parts of an application. In this case, we want to use them to authenticate the user and their privileges within a video call. 
+
+The token is generated based on the HMAC-SHA256 cryptographic algorithm. This algorithm is an industry standard for authentication. 
```

Comment:
Consider combining these sentences with the next two. Reduce repetitiveness of the word `algorithm` while conveying a clearer connection of "what" is a token:

I.e:
> The token is generated based on the HMAC-SHA256 cryptographic algorithm. We don't need to dive into cybersecurity theory; just know that it's an industry standard for authentication and it will give you a code that verifies your user's identity. In Node we can generate this token using the `agora-token` package.